iree/turbine/aot/support/ir_utils.py

```diff
@@ -150,6 +151,50 @@ def infer_external_from_tensor(
         return bool(self.external), self.external_scope, None
 
 
+###############################################################################
+# Reference Trackers - ad-hoc solution to the memory leaks
```

Comment:
Can you add a small description about why this RefTracker is here and what makes it different from the `torch.RefTracker` implementation?